### PR TITLE
add some functions

### PR DESCRIPTION
**Pull request by Mark3K**

### CODE REVIEW

**Comment by favoretti** [comment]
Hey,

Thanks for the changes. However, your PR has conflicts, would you be so kind to rebase?
Thanks!

Vlad

File: nsnitro/nsresources/nsgslbvserver.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+        """
+        self.options['servicename'] = servicename
+
+    def get_servicename(self):
+        """
+        """
+        return self.options['servicename']
+
+    # Operations methods
+    '''
```
Comment (favoretti) [inline nsnitro/nsresources/nsgslbvserver.py]: It seems you left some stuff commented out. Not intentional I guess?

File: nsnitro/nsresources/nsgslbvserver.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        Use this API to update csvserver of a given name.
+        """
+        __gslbvs = NSGSLBVServer()
+        __gslbvs.set_name(gslbvs.get_name())
+        __gslbvs.set_servicename(gslbvs.get_servicename())
+        __gslbvs.set_weight(gslbvs.get_weight())
+
+        return __gslbvs.update_resource(nitro)
+
+    '''
```
Comment (favoretti) [inline nsnitro/nsresources/nsgslbvserver.py]: Same here.

File: nsnitro/nsresources/nscsvserverstat.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+class NSCSVServerStat(NSBaseResource):
+    
+    def __init__(self, json_data=None):
+        super(NSCSVServerStat, self).__init__()
+        self.options = {
+            'name': '',                 # Name of the content switching virtual server for which to display statistics. \
+                                        # To display statistics for all configured Content Switching virtual servers, \
+                                        # do not specify a value for this parameter.
+            'clearstats': '',
+            # ----- read-onlu options
```
Comment (favoretti) [inline nsnitro/nsresources/nscsvserverstat.py]: onlu -> only ;)

**Comment by Mark3K** [comment]
Hi,

Thanks for your advice. I corrected the problems you mentioned above and rebased my branch.
Thanks for your time!

**Comment by favoretti** [comment]
Hmm, I'm not sure how you rebased it, but GitHub fails to auto-merge it. I'll try to look into it today to rebase myself and merge. Otherwise feel free to give it another look.

**Comment by Mark3K** [comment]
Oh, I am sorry for this, I googled and rebased my branch again, it seems OK this time.

**Comment by favoretti** [comment]
Thanks! Merged :)

**Pull request merged**